lib/ipaddr.rb

```diff
@@ -110,8 +110,13 @@ def self.new_ntoh(addr)
 
   # Convert a network byte ordered string form of an IP address into
   # human readable form.
+  # It expects the string to be encoded in Encoding::US_ASCII (BINARY).
```

Comment:
Encoding::US_ASCII is different from ASCII_8BIT.  US_ASCII is a 7-bit encoding where codes above 127 cannot be contained.

Reply:
copy&pasted wrong string 😓